cmd/satellite/Dockerfile

```diff
@@ -5,17 +5,16 @@ FROM brimstone/golang as build-env
 
 # final stage
 FROM alpine
-ENV REDIS_ADDR=redis:6379 \
-    REDIS_PASSWORD= \
-    REDIS_DB=0 \
+ENV API_KEY= \
+    CONF_PATH=/root/.storj/satellite/config.yaml \
+    OVERLAY_URL=redis://redis:6379/?db=0 \
     IDENTITY_ADDR=:7777 \
-    HTTP_PORT=:8080 \
-    BOOTSTRAP_ADDR=bootstrap.storj.io:8080
+    BOOTSTRAP_ADDR=bootstrap.storj.io:8080 \
```

Comment:
We should probably update this to satellite.storj.io:7776

Reply:
We can update that later when we get a satellite there.

Reply:
Sounds good.